### ISSUE TITLE
Add the ability to filter ammo chart by my trader levels or price

### ISSUE THREAD

**Issue by Reithan** [kokarn/tarkov-tools#155]
Since tarkov tools has my trader levels set in settings, and it has market data from it's API, it would be awesome to be able to set a max price, or max trader level for ammos and be able to see what ammos I should be considering.

**Comment by kokarn** [comment]
Wow, that's a great suggestion! Can't believe I haven't thought of this sooner.

**Comment by AnyFinCanHappen** [comment]
Hello @kokarn can I work on this issue? Unless @Reithan is already working on this one.

Also, some UI suggestions. Since we are having multiple filter categories, we can have a `VictoryLegend` instance for each category. So for our case we would have 3 legends, "Filter by caliber", "Filter by Trader Level", and "Filter by Price".

If we go ahead with using three `VictoryLegend` instances, then we could have the legends be more horizontally shaped and  ordered to be on top of the graph.

Mock: 
The three red boxes represent the legends
![tarkov_tool_idea](https://user-images.githubusercontent.com/44888851/154381518-d0fb186f-b305-4089-9d84-c92a881112d7.png)

Let me know what you think!

**Comment by Reithan** [comment]
Oh, as to the question if I'm currently working on this, no I am no, @AnyFinCanHappen

**Comment by AnyFinCanHappen** [comment]
@Reithan  thanks for the feedback,

Im fine with leaving "filter by caliber" as it is. If we do that I'm not really sure where to put the filter by traders and price range. So for now I just work on the functionality for each of the filters and wait on @kokarn opinion on where we should put the filters.

Also some things to note: I'm not sure if we can have custom components in a `VictoryContainer` So we may not be able to place the price range and trader filter along side the caliber filter.

**Comment by kokarn** [comment]
Very cool that you want to work on this @AnyFinCanHappen !

I'n general I'm fine with you trying out what you think works best.

If you want my thinking I was thinking more about using the `Filter` that already exists and putting on top. I've had trouble styling the Legend in the past and I think people want to use all filters combined in the same way they do on other pages.

**Comment by Reithan** [comment]
Wanted to revisit one thing I said:
I said min-max price range, but tbh, I can't think of a super solid reason to want to filter to a min price? Maybe max price by itself would be enough?
I could definitely be wrong though. If you think min would be valuable, it shouldn't be too hard to take or leave it when/if you add max price.

**Comment by kokarn** [comment]
I agree, no need for "min" price which also makes the implementation match the rest of the site 👍

**Comment by kokarn** [comment]
No worries at all! Stuff happens all the time :) Just let me know if you need help and/or support with anything, whenever you get the opportunity :)
